README.md

```diff
@@ -0,0 +1,90 @@
+# Project Name
+<!--- 
+If you want to use badges, then put them first. These are just some examples. See https://shields.io for others or to customize this set of shields. You might want to include dependencies, project status and licence info here.
+--->
+![GitHub stars](https://img.shields.io/github/stars/scottydocs/README-template.md?style=social)
+![GitHub forks](https://img.shields.io/github/forks/scottydocs/README-template.md?style=social)
+
+<!--
+This section should contain a brief description that should clearly state what the project does in no more than 3 sentences. Don't go overboard, people won't read it. 
+
+Below are some good examples from other projects that you can use to help you out.
+
+  TensorFlow makes it easy for beginners and experts to create machine learning models. See the sections below to get started. (from Tensorflow homepage)
+
+  An open source machine learning framework that accelerates the path from research prototyping to production deployment. (from PyTorch homepage)
+
+  Kubernetes is an open-source system for automating deployment, scaling, and management of containerized applications. (from Kubernetes homepage)
+-->
+
+<project_name> is a <utility/tool/feature> that allows <insert_target_audience> to do <action/task_it_does>.
+
+Additional line of information text about what the project does.
+
+## Prerequisites
+<!--- 
+This section should include information about platform support and any prerequistes that a user will need to be able to compile and run the project.
+
+If you are not sure what specifically to put, use the below boilerplate and complete/delete as appropriate.
+--->
+Before you begin, ensure you have met the following requirements:
+* You have a `<Windows/Linux/Mac>` machine. State which OSs are supported and which are not.
+* You have installed the latest/specific version of `<coding_language/dependency/requirement_1>`.
+* You have read `<guide/link/documentation_related_to_project>`.
+
+## Quick Start
```

Comment:
I wasn't sure about if this should be split into 2 sections, like 'Installation' and 'Usage' sections which seems a popular convention whilst I was researching, so opinions/thoughts welcome.